stories/10-For Developers/SampleCode/Layout.stories.js

```diff
@@ -0,0 +1,129 @@
+import {compile} from 'ember-source/dist/ember-template-compiler';
+
+export default {title: 'For Developers/Sample Code/Layouts'};
```

Comment:
We've been leaning towards including our stories in `.mdx` files. This gives us more control over the layout of the docs pages. It's a little more verbose but I'd like to keep this consistent across the docs. There are a handful of ways to include stories with storybook and I'd prefer if we only use one of them.

There's an example here and we can go over this in more detail together if you want.
https://nypr-design-system.nypr.digital/latest/index.html?path=/docs/for-developers-documenting-components--page#creating-a-story-file